copier.yml

```diff
@@ -38,41 +38,36 @@ use_ingestion:
 
 ingestion_framework:
     type: str
-    when: "{% if use_ingestion %}true{% endif %}"
     help: What ingestion framework would you like to use?
     choices:
         Airbyte: airbyte
     default: airbyte
 
 destination_id_dev:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Destination Id for dev instance
 
 source_id_dev:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Source Id for dev instance
 
 workspace_id_dev:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Workspace id for dev instance
+    default: a7e79db0-7f4f-4663-94db-ba392860fa90
 
 destination_id_prod:
     type: str
-    when: "{% if use_ingestion and ingestion_framework == 'airbyte' %}true{% endif %}"
     help: Destination Id for prod instance
```

Comment:
Maybe there could be and default as None? then if someone doesn't need it just click enter.

Reply:
Should this apply to all prod ingestion questions?